javascripts/testimony.js

```diff
@@ -1,26 +1,59 @@
+		document.getElementById("reviewFourth").innerHTML = "<i>One of the most underrated Apps in the store. I wish the ratings system went to 10; this is the best app for Bible students. My hat is off to the developers. Some of the best translations and commentaries that currently exist are here ... for free. How are all the reviews not 5 stars?</i> <br/> <b>-J.F., AndBible user</b> <br/> <span class='fa fa-star checked'></span><span class='fa fa-star checked'></span><span class='fa fa-star checked'></span><span class='fa fa-star checked'></span><span class='fa fa-star checked'></span> <small>(Google Play)</small>";
+	}
+	if (randomNumber == 4) {
+		document.getElementById("reviewFirst").innerHTML = "<div class='feature-testimonial'><span class='badge badge-success'>Feature Review</span><br/><i>I LOVE And Bible. It is the best Bible app out there. I have Logos and Olive Tree as well for a few copyright resources, but And Bible is my daily Bible.</i> <br/> <b>Jared Hood, OT Lecturer, Academic Dean: Presbyterian Theological College <br/> Executive Editor: Reformed Theological Review</b><br/> </div><hr/>";
+		document.getElementById("reviewSecond").innerHTML = "<i>It's nice that they only use 'legal' Bibles. A very good app; it's fast and easy to use. In no other application had I found the Reina Valera 1909 Bible with italic letters, as in the printed one and with Strong's numbers. It has been very useful, thank you. Blessings!</i> <br/> <b>-H.W., AndBible user</b><br/> <span class='fa fa-star checked'></span><span class='fa fa-star checked'></span><span class='fa fa-star checked'></span><span class='fa fa-star checked'></span><span class='fa fa-star checked'></span> <small>(Google Play)</small><hr/>";
+		document.getElementById("reviewThird").innerHTML = "<i>The 'Bible Study App' (And Bible) for Android is so good that when I'm studying the Bible and I use desktop software I often find myself going back to my Android device because of ease of use and the many features that it has. It helps me to easily organize my thoughts with labels and study pads. The best Bible app for mobile and tablets, bar none!</i> <br/> <b>-T.B., AndBible user</b> <br/> <span class='fa fa-star checked'></span><span class='fa fa-star checked'></span><span class='fa fa-star checked'></span><span class='fa fa-star checked'></span><span class='fa fa-star checked'></span> <small>(Google Play)</small><hr/>";
+		document.getElementById("reviewFourth").innerHTML = "<i>Excellent application, the best. You can see different versions, dictionaries in Greek and Hebrew (to English) and biblical commentaries ... it's like having the study library in your pocket. Thanks! <br/> <b>-A.B., AndBible user</b> <br/> <span class='fa fa-star checked'></span><span class='fa fa-star checked'></span><span class='fa fa-star checked'></span><span class='fa fa-star checked'></span><span class='fa fa-star checked'></span> <small>(Google Play)</small>";
+	}
```

Comment:
Well, we can start with this, but I'd like to get this code abit more generic si tgat ut us easy to add more items to our review database.

I.e.

We have array of reviews:

```js
var reviews = [
{
 content: "content of review", // no need to have html there even.
 by: "name of reviewer or initials",
 stars: 5, // default, no need to specify to every item
 source: "Google Play",  // default, no need to specify to every item
},
...
];
```

and code would randomly pick 5 items from this list and render them nicely to the page.

Reply:
My idea is that we will go through our Google Play reviews and add *a lot* more of those here. Tens, even hundreds of 5-star reviews. This approach is not sustainable then any more.

Reply:
Do you think this is doable to you? If not I could offer to help.